messenger/http_transporter.go

```diff
@@ -44,6 +44,18 @@ var (
 	errNotStarted      = errors.New("HTTP transport has not been started")
 	errTerminal        = errors.New("HTTP transport is terminated")
 	errAlreadyRunning  = errors.New("HTTP transport is already running")
+
+	httpTransport, httpClient = &http.Transport{
```

Comment:
Are you positive this doesn't work as two declarations? I could sworn I'd done that before for this exact reason.